common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RemoteBlockPushResolver.java

```diff
@@ -95,6 +96,12 @@ public class RemoteBlockPushResolver implements MergedShuffleFileManager {
   public static final String MERGE_DIR_KEY = "mergeDir";
   public static final String ATTEMPT_ID_KEY = "attemptId";
   private static final int UNDEFINED_ATTEMPT_ID = -1;
+
+  /**
+   * The flag for deleting the current merged shuffle data.
+   */
+  public static final int DELETE_CURRENT_MERGED_SHUFFLE_ID = -1;
```

Comment:
nit: Rename this as `DELETE_ALL_MERGED_SHUFFLE` or some such ?
That is the behavior we actually are looking for - in case the implementation evolves.